drevo/views/interview_and_proposal.py

```diff
@@ -23,6 +24,12 @@ def my_interview(request, id):
                 context['activity'] = [i for i in context['sections'] if i.startswith('Мои') or
                                        i.startswith('Моя')]
                 context['link'] = 'users:myprofile'
+                invite_count = len(FriendsInviteTerm.objects.filter(recipient=user.id))
```

Comment:
Лучше использовать count(), так как он работает быстрее (не выполняет загрузку данных из БД).

```
A count() call performs a SELECT COUNT(*) behind the scenes, so you should always use count() rather than loading all of the record into Python objects and calling len() on the result (unless you need to load the objects into memory anyway, in which case len() will be faster).
```
https://docs.djangoproject.com/en/4.2/ref/models/querysets/#count